plugins/kubernetes/app/models/kubernetes/release.rb

```diff
@@ -14,9 +15,9 @@ class Release < ActiveRecord::Base
     validates :status, inclusion: STATUSES
     validate :docker_image_in_registry?, on: :create
 
-    STATUSES.each do |s|
-      define_method("#{s}?") { status == s }
-    end
+    scope :not_dead, -> { where.not(status: :dead) }
+    scope :excluding, ->(ids) { where.not(id: ids) }
+    scope :with_not_dead_release_docs, -> { joins(:release_docs).where.not(Kubernetes::ReleaseDoc.table_name => { status: :dead }) }
```

Comment:
I'm not yet sure where this is used, but as a precaution, we also have a `failed` status, does it qualify as `dead` as well in this context?